instagram/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import './CommentSection.css';
+import PropTypes from 'prop-types';
+
+const CommentSection = props => {
+    return(
+        <div className="comment-container">
+            <p><span>{props.data.username}</span> {props.data.text}</p>
```

Comment:
Imagine how much cleaner this could be with [object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring):
```javascript
const CommentSection = ({ data }) => {
  const { username, text } = data;
  return (
    <div className="comment-container">
      <p>
        <span>{username}</span>
        {text}
      </p>
    </div>
  );
};
```